code/api/src/config/database.json

```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "username": "isabellevillasenor",
+    "username": "rachelbuchta",
```

Comment:
Should this have been added to .gitignore?

Reply:
It should! We'll have to `rm` it since it's already been tracked on GitHub and then add it to the gitignore